dev_guide/managing_images.adoc

```diff
@@ -0,0 +1,582 @@
+similar to a Docker image repository.
+
+By watching an image stream, builds and deployments can receive notifications
+when new images are added and react by performing a build or deployment,
+respectively.
+
+There are many ways you can interact with images and set up image streams,
+depending on where the images' registries are located, any authentication
+requirements around those registries, and how you want your builds and
+deployments to behave. The following sections cover a range of these topics.
+
+[[tagging-images]]
+== Tagging Images
+
+Before working with image streams and their tags, it will help to first
+understand image tags in the context of Docker generally.
+
+Docker images can have names added to them that make it more intuitive what they
+contain. When interacting directly with images using the `docker` CLI, the
+`docker tag` command essentially adds an alias to an image that can consist of
+several parts. Those parts can include:
+
+----
+<registry_host>/<user_name>/<image_name>:<tag>
+----
+
+The `<user_name>` part in the above could also refer to a
+link:../architecture/core_concepts/projects_and_users.html#projects[project] or
+link:../architecture/core_concepts/projects_and_users.html#namespaces[namespace]
+if the image is being stored in an {product-title} or Kubernetes environment.
+
+{product-title} provides the `oc tag` command, which is similar to the `docker
+tag` command, but operates on image streams instead of directly on images.
+
+[NOTE]
+====
+See Red Hat Enterprise Linux 7's
+https://access.redhat.com/documentation/en/red-hat-enterprise-linux-atomic-host/version-7/getting-started-with-containers/#creating_docker_images[Getting
+Started with Containers] documentation for more about tagging images directly
+using the `docker` CLI.
+====
+
+[[adding-tag]]
+=== Adding Tags to Image Streams
+
+Keeping in mind that an image stream in {product-title} comprises one or more
+Docker images identified by tags, an image stream can also be configured so that
+a tag "tracks" another one.
+
+For example, to configure the *ruby* image's *latest* tag to always refer to the
+current image for the tag *2.0*:
+
+====
+----
+$ oc tag ruby:latest ruby:2.0
+----
+====
+
+[[tag-removal]]
+=== Removing Tags from Image Streams
+
+You can also stop tracking a tag by removing the tag. For example, to stop
+tracking an existing *latest* tag:
+
+====
+----
+$ oc tag -d ruby:latest
+----
+====
+
+However, while the above command removes the tag from the image stream
+definition, it does not remove it from the image stream status. The image stream
+definition is user-defined, whereas the image stream status reflects the
+information the system has from the specification.
+
+To remove a tag completely from an image stream:
+
+====
+----
+$ oc delete istag/ruby:latest
+----
+====
+
+[[referencing-images-in-image-streams]]
+== Referencing Images in Image Streams
```

Comment:
Right, totally forgot about this part. Hmm... how about merging the two parts and specifying all possible options at once with links to each where details will be provided?